### DISCUSSION TITLE
How to download CHIRPS data

### DISCUSSION THREAD

**Discussion by destaw2112** [jddingemanse/awtiCodeDev#39] [Q&A]
How can I extract 24 years of CHIRP rainfall data products in the Abaya-Chamo Subbasin?
Thank you!

**Comment by jddingemanse** [comment]
@Demiso1 should be able to help you; he wrote code for this. We wrote some code for downloading CHIRPS data, you can find a bit of text about it through [this link](https://github.com/jddingemanse/awtiCodeDev#using-the-package-awticode).

**Comment by destaw2112** [comment]
Dear Jddingemanse/Awticodedev

Thank you so much!
 Best Regards
   Destaw Akili (MSc.)
  Lecturer in  Water Resources and Irrigation Engineering
   Arba Minch Water Technology Institute (AWTI)
  Arba Minch University, Ethiopia
